spec/configurable_spec.rb

```diff
@@ -42,6 +42,57 @@ module Xcake
       end
     end
 
+    context "when creating configurations for the first time" do
+
+      context "creating a debug configuration" do
+
+        it "should create a configuration if a name is specified" do
+          configuration = @configurable.debug_configuration(:debug)
+          expect(@configurable.debug_configuration(:debug)).to eq(configuration)
+        end
+
+        it "should name the configuration if a name is specified" do
+          configuration = @configurable.debug_configuration(:debug)
+          expect(configuration.name).to eq('debug')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.